jest-setup.js

```diff
@@ -8,3 +8,19 @@ global.TextDecoder = TextDecoder;
 import { matchers } from './src/matchers';
 
 expect.extend(matchers);
+
+window.MessageChannel = jest.fn().mockImplementation(() => {
+  let onmessage;
+  return {
+    port1: {
+      set onmessage(cb) {
+        onmessage = cb;
+      },
+    },
+    port2: {
+      postMessage: (data) => {
+        onmessage?.({ data });
+      },
+    },
+  };
+});
```

Comment:
is this still needed if react is pinned to v18?

Reply:
Good catch, done!